packages/app/src/App.js

```diff
@@ -2,14 +2,19 @@ import React from "react";
 import logo from "./logo.svg";
 import GoogleBtn from "./components/GoogleBtn/GoogleBtn";
 import "./App.css";
+import FamilyMemberInfo from "./components/familyInformation/familyInformation";
+import family_info from "./components/familyInformation/dummyData";
 
 function App() {
   return (
     <div className="App">
       <div>
         <GoogleBtn />
+        <div style={{ width: "40vw", margin: "0 auto" }}>
```

Comment:
instead of having this live on the home page

can we bring in ReactRouter and have it live on the Application Details Page? 
https://reactrouter.com/web/guides/quick-start

Can create another ticket for that if neeed